src/matrixrtc/NewMembershipManager.ts

```diff
@@ -276,9 +306,23 @@ export class MembershipManager implements IMembershipManager {
         if (deviceId === null) throw Error("Missing deviceId in client");
         this.deviceId = deviceId;
         this.stateKey = this.makeMembershipStateKey(userId, deviceId);
+        this.state = MembershipManager.defaultState;
     }
 
-    // Membership Event parameters:
+    // MembershipManager mutable state.
+    public state: ActionSchedulerState;
```

Comment:
Does this need to be public?

```suggestion
    private state: ActionSchedulerState;
```

Reply:
I was considering using this to simplify the useECConnection state hook in EC